### PR TITLE
KEYCLOAK-10376 Improve load balance and proxy documentation

### PR DESCRIPTION
**Pull request by andymunro**
@tkyjovsk This includes the updated load balance graphic for your review.  If you think we need more updates to the text, we can continue on this in January.

### CODE REVIEW

File: server_installation/topics/clustering/load-balancer.adoc [hunk 1]
```diff
@@ -5,6 +5,11 @@ This section discusses a number of things you need to configure before you can p
 in front of your clustered {project_name} deployment.  It also covers configuring the built in load balancer that
 was <<_clustered-domain-example, Clustered Domain Example>>.
 
+The following diagram illustrates the use of a load balancer between clients and a {project_name} cluster. In another scenario, the load balancer could be a reverse proxy server.
```
Comment (tkyjovsk) [inline server_installation/topics/clustering/load-balancer.adoc]: Perhaps a better formulation would be:

"The following diagram illustrates the use of a load balancer. The load balancer serves as a reverse proxy between the clients and the {project_name} servers."

File: server_installation/topics/clustering/load-balancer.adoc [hunk 2]
```diff
@@ -5,6 +5,11 @@ This section discusses a number of things you need to configure before you can p
...
+
+[[load-balancer-diagram]]
+.Example Load Balancer Diagram
+image:{project_images}/load_balancer.png[]
```
Comment (tkyjovsk) [inline server_installation/topics/clustering/load-balancer.adoc:12]: Regarding the diagram, see my latest comment in JIRA: https://issues.redhat.com/browse/KEYCLOAK-10376?focusedCommentId=13937113&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13937113

File: server_installation/topics/clustering/load-balancer.adoc (hunk 1 above)
Comment (andymunro) [reply server_installation/topics/clustering/load-balancer.adoc]: Thanks, @tkyjovsk.  I have made that change.

**Comment by andymunro** [comment]
@stianst I added SVGs in the rhsso-images and keycloak-images folders.  Does this look good to go?

**Comment by andymunro** [comment]
@stianst This was completed some time ago.  Can we add this for 7.4.1?  It's had multiple reviews.

**Comment by andymunro** [comment]
@hmlnarik Can you approve this?

**Comment by andymunro** [comment]
@pskopek Can you approve this so we can include this for 7.4.1?

**Review by tkyjovsk — APPROVED** [review]
LGTM.

**Comment by andymunro** [comment]
@pskopek Can you approve this change which has been around a long time?

**Comment by andymunro** [comment]
@sguilhen merged this in downstream https://github.com/keycloak/keycloak-documentation-prod/pull/168.
@abstractj can you merge it in upstream

**Comment by sguilhen** [comment]
Why has this been closed?

**Comment by andymunro** [comment]
@sguilhen Sorry that was my mistake. It was one of 3 that were merged in downstream, but not merged in upstream.  Can you reopen it.

**Pull request merged**